apollo-portal/src/main/resources/static/scripts/controller/NamespaceController.js

```diff
@@ -94,32 +94,38 @@ namespace_module.controller("LinkNamespaceController",
                 selectedClusters = data;
             };
             $scope.createNamespace = function () {
-                if ($scope.type == 'link') {
-                    if (selectedClusters.length == 0) {
+                if ($scope.type === 'link') {
+                    if (selectedClusters.length === 0) {
                         toastr.warning($translate.instant('Namespace.PleaseChooseCluster'));
                         return;
                     }
 
-                    if ($scope.namespaceType == 1) {
-                        var selectedNamespaceName = $('#namespaces').select2('data')[0].id;
-                        if (!selectedNamespaceName) {
+                    if ($scope.namespaceType === 1) {
+                        var selectedNamespaceNames = $('#namespaces').select2('data');
+                        var ids = []
+                        selectedNamespaceNames.map().forEach(function (namespace) {
```

Comment:
`selectedNamespaceNames.map()` will throw the following exception：
```js
TypeError: undefined is not a function
    at Array.map (<anonymous>)
    at $scope.createNamespace (NamespaceController.js:110:48)
    at fn (eval at compile (angular.min.js:224:319), <anonymous>:4:236)
    at b (angular.min.js:121:504)
    at e (angular.min.js:266:74)
    at b.$eval (angular.min.js:140:467)
    at b.$apply (angular.min.js:141:188)
    at HTMLFormElement.<anonymous> (angular.min.js:266:126)
    at Lf (angular.min.js:36:394)
    at HTMLFormElement.d (angular.min.js:36:341)
```
An array can be iterated as follows:
```js
angular.forEach(selectedNamespaceNames, function(item){
                            ids.push(item.id)
                        });
```

Reply:
i am so sorry, my local env have some trouble